### DISCUSSION TITLE
My document's photo

### DISCUSSION THREAD

**Discussion by GabrielFMontoni** [community/community#54306] [GitHub Education]
### Hi everyone,

I can't join to the benefits of github education because i cant send my photo, im studying software engineering at FIAP, but its a distance course, then my documents are online, but when i send this, the system tell : your photo is modificated and i cant proceed. What should i do?

**Comment by hrishikesh-kocharekar-dev** [comment]
If you are having difficulty submitting a photo for verification in the GitHub Education program, I recommend contacting GitHub Education support directly to explain your situation and ask for assistance.

You can contact GitHub Education support by submitting a support request here: https://support.github.com/contact/education. In your message, be sure to provide a clear explanation of your situation, including the fact that you are enrolled in a distance learning program and that your documents are online. You may also want to include any additional documentation or proof of enrollment that you have available.
